docker/uwsgi/index.html

```diff
@@ -1,73 +1,93 @@
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-91465549-9"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'UA-91465549-9');
+  </script>
+  <style>
+    img.grayscale {
+      filter: url("data:image/svg+xml;utf8,<svg xmlns=\'http://www.w3.org/2000/svg\'><filter id=\'grayscale\'><feColorMatrix type=\'matrix\' values=\'0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0\'/></filter></svg>#grayscale"); /* Firefox 3.5+ */
+      filter: gray; /* IE6-9 */
+      -webkit-filter: grayscale(100%); /* Chrome 19+ & Safari 6+ */
+    }
+    img.grayscale:hover {
+      filter: none;
+      -webkit-filter: grayscale(0%);
+    }
+    a.btn.btn-primary {
+      background: #a3a3a3;
+      border-color: #a3a3a3;
+      color: #fff;
+    }
+    a.btn.btn-primary:hover {
+      background: #777777;
+      border-color: #707070;
+      color: #fff;
+    }
+    p {
+      font-size: 14px;
+      margin-bottom: 14px;
+    }
+    h5 {
+      margin-bottom: 5px;
+    }
+    h3 {
+      color: #50596C;
+    }
+    img {
+      margin-top: 10px;
+    }
+  </style>
 </head>
 <body>
 
-<div class="container" id="content">
+<div class="container" style="margin-top: 50px;" id="content">
   <div class="columns">
-    <div class="column col-6 col-mx-auto">
-        <h1>µRaiden demo</h1>
+    <div class="column col-9 col-mx-auto">
+        <h1>µRaiden Demos</h1>
     </div>
   </div>
-
   <div class="columns">
-  <div class="column col-6 col-mx-auto" id="description">
+  <div style="margin-top: 10px" class="column col-9 col-mx-auto" id="description">
     <h5>1. Install Web3 browser extension</h5>
-    <p>This allows you to access the Ethereum networks. Use <a href="https://metamask.io/">Metamask</a> or <a href="https://chrome.google.com/webstore/detail/parity-ethereum-integrati/himekenlppkgeaoeddcliojfddemadig">Parity</a> - works in Chrome and Firefox</p>
-    <h5>2. Get some Ropsten Testnet Ether (For free!)</h5>
-    <p>Try <a href="http://faucet.ropsten.be:3001/">some</a> <a href="https://faucet.metamask.io/">faucets</a></p>
-    <h5>3. Buy µRaiden demo tokens with your ether</h5>
-    <p>You can buy some when you first access the resource.</p>
-    <h5>4. Access the content</h5>
-    <p>And thanks for coming!</p>
+    <p><a style="color: #a3a3a3" href="https://metamask.io/">Metamask</a> and <a style="color: #a3a3a3" href="https://chrome.google.com/webstore/detail/parity-ethereum-integrati/himekenlppkgeaoeddcliojfddemadig">Parity</a> allow you to access the Ethereum DApps, in both Chrome and Firefox.</p>
+    <h5>2. Get some Kovan Testnet Ether</h5>
```

Comment:
We're not using Kovan anymore, as Infura's nodes are very unstable. We use Ropsten only, previous version had some faucet links you may want to keep in the page.

Reply:
Still Kovan